### DISCUSSION TITLE
8.0 AIO Release?

### DISCUSSION THREAD

**Discussion by johnnyq** [nextcloud/all-in-one#4393] [Questions] [answered]
Hello this is basic question and in no hurry, we planned on doing Nextcloud AIO upgrades for our clients tonight but we noticed that it has been 7 days and no release of 8.0. Would it be safe to assume that 8.0 will not happen today? and that we should plan for next Friday?

**Comment by szaimen** [comment, marked as answer]
Hi, the overview always mentiones the date for when the release is planned. In this case see https://github.com/nextcloud/all-in-one/issues/4317

**Comment by johnnyq** [comment]
Thanks @szaimen =]
